src/utils/configs.js

```diff
@@ -46,18 +46,23 @@ if (window.APP_CONFIG) {
     style.innerHTML = `:root{${colorVars.join("\n")}}`;
     document.head.prepend(style);
   }
+} else {
+  configs.APP_CONFIG = {};
 }
 
 const isLocalDevelopment = process.env.NODE_ENV === "development";
 
+const enableAllBlacklist = ["public_rooms"];
 configs.feature = featureName => {
-  const enableAll = isLocalDevelopment && !process.env.USE_FEATURE_CONFIG;
-
-  const features = configs.APP_CONFIG && configs.APP_CONFIG.features;
-
-  const forceEnableSpoke = featureName === "enable_spoke" && isAdmin;
-
-  return forceEnableSpoke || enableAll || (features && features[featureName]);
+  const value = configs.APP_CONFIG.features[featureName];
+  if (typeof value === "boolean") {
+    const enableAll =
+      isLocalDevelopment && !process.env.USE_FEATURE_CONFIG && !enableAllBlacklist.includes(featureName);
+    const forceEnableSpoke = featureName === "enable_spoke" && isAdmin;
```

Comment:
see above: the check around `public_rooms` likely should be made a permission check, and logic surrounding it in the client should be removed.

in general, it seems like a anti-pattern for us to be checking `isAdmin` anywhere in the client insofar as that flag confers some rights on the server. (it still should be sane to check a users admin status in the client, when that admin status can't or isn't involved in privileges but "UI convention")